Apps/W1/HybridGP/app/src/Migration/Vendors/GPVendorMigrator.codeunit.al

```diff
@@ -455,4 +455,79 @@ codeunit 4022 "GP Vendor Migrator"
             Vendor.Modify(true);
         end;
     end;
+
+    procedure MigrateVendorClasses()
+    var
+        GPCompanyAdditionalSettings: Record "GP Company Additional Settings";
+        GPPM00200: Record "GP PM00200";
+        GPPM00100: Record "GP PM00100";
+        VendorPostingGroup: Record "Vendor Posting Group";
+        Vendor: Record Vendor;
+        HelperFunctions: Codeunit "Helper Functions";
+        ClassId: Text[20];
+        AccountNumber: Code[20];
+        MigrateVendorClasses: Boolean;
+    begin
+        MigrateVendorClasses := GPCompanyAdditionalSettings.GetMigrateVendorClasses();
+
+        if not MigrateVendorClasses or not GPPM00200.FindSet() then
```

Comment:
It is a good practice to split IFs if there is a transaction within it.
AL will execute both conditions. So it could be better to extract Findset to a separate If statement.